BuildOptimizedImages/src/Web/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM microsoft/aspnetcore-build:1.0.1
+
+RUN mkdir /app
+WORKDIR /app
+
+ADD project.json .
+RUN dotnet restore --no-cache
```

Comment:
It's because I had misunderstood what the flag did: https://github.com/aspnet/MusicStore/pull/713#discussion_r84135005